examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -294,17 +293,12 @@ class _AllSectionsView extends AnimatedWidget {
     // The layout's progress from from a column to a row. Its value is
     // 0.0 when size.height equals the maxHeight, 1.0 when the size.height
     // equals the midHeight.
-    final double tColumnToRow =
-      1.0 - ((size.height - midHeight) /
-             (maxHeight - midHeight)).clamp(0.0, 1.0);
-
+    final double tColumnToRow = 1.0 - ((size.height - midHeight) / (maxHeight - midHeight)).clamp(0.0, 1.0);
```

Comment:
the newline after the `/` is pretty important here as it visualizes the division.

I could imagine:
```dart
    final double tColumnToRow = 1.0 - ((size.height - midHeight) /
                                       (maxHeight - midHeight)).clamp(0.0, 1.0);
```
...or maybe even better:

```dart
    final double tColumnToRow = 1.0 - ((size.height - midHeight) /
                                       (maxHeight   - midHeight)).clamp(0.0, 1.0);
```

Reply:
Big brother is watching you :D https://github.com/flutter/flutter/pull/25251/files#r241495408

This kind of alignment is why I implemented `// @formatter:off`. It seems really hard to catch this pattern and automatically format it.